include/default-opts/lwipopts.h

```diff
@@ -29,6 +29,12 @@
 #define MSCHAP_SUPPORT 1
 #define LWIP_TIMEVAL_PRIVATE 0
 
+#ifdef __CPU_GR740
+#define CHECKSUM_GEN_TCP 0
+#define CHECKSUM_GEN_UDP 0
+#define CHECKSUM_GEN_IP  0
+#endif
```

Comment:
The default options should remain unchanged, please add `lwipopts.h` to `phoenix-rtos-project/_projects/sparcv8leon-gr740-mini/lwip/` directory and set `export LWIPOPTS_DIR="$(pwd)/_projects/sparcv8leon-gr740-mini/lwip"` in the `build.project` for gr740-mini target - see e.g. https://github.com/phoenix-rtos/phoenix-rtos-project/blob/9d39cf340b7678e584882d2ac3b5f8f99e700b0e/_projects/ia32-generic-qemu/build.project#L14 for reference